### ISSUE TITLE
Tab lists should only occupy a single tab stop

### ISSUE THREAD

**Issue by SaschaCowley** [microsoft/vscode#106438]
- VSCode Version: 1.49.0 (e790b931385d72cf5669fcefc51cdf65990efa5d)
- OS Version: Windows 10.0.19041.450x64

Steps to Reproduce:

1. Use `F6` or `Tab` to move to somewhere tabbed (the activity switcher or file tabs, for example).
2. Try to use `Tab` to get out of these areas.

The expected behaviour is that tabbed areas  only occupy 1 position in the tab order, and are navigable with the arrow keys.

The actual behaviour is that each tab occupies its own position in the tab order.

I think an exception can/should be made for the file tabs in that the currently focused (not necessarily selected) tab's controls may come after it in the tab order (however, per #106441, they should only occupy a single tab stop).

See [Example of Tabs with Automatic Activation](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) and [Example of Tabs with Manual Activation](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html).

Does this issue occur when all extensions are disabled?: Yes/No
Yes.

**Comment by isidorn** [comment]
@SaschaCowley I think this makes sense, however let's first discuss https://github.com/microsoft/vscode/issues/106441 beore we do anything here.

**Comment by MarcoZehe** [comment]
What else needs to be done here? As far as I can see, these tab lists now behave in a similar way as other toolbars, after #106441 got closed. Left and Right moves between them, Space toggles their expanded/collapsed state, and tab moves to the next container or the associated panel.

**Comment by isidorn** [comment]
Activity bar now works nicely. What still needs to change is the `tablist` where we keep all the tabs. @MarcoZehe maybe you have tabs disabled?

Currently each tab and each close button inside the tab are a tabstop. And the arrow navigation works nicely - yeey.

What @SaschaCowley suggests and I agree:
* tablist should occupy only one tab stop just like other toolbars, and users should use left right to navigate between tabs

I will look into making this change today.
fyi @bpasero

**Comment by bpasero** [comment]
Makes sense to me to tab into tabs-stripe and navigate within via arrow keys.

**Comment by isidorn** [comment]
I did the changes. This works nicely for me now.
@bpasero try it out and check out the code changes if you would like and let me know what you think
